src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -168,8 +168,7 @@ internal static class LogMessages
         public const string IDX10672 = "IDX10672: GetKeyedHashAlgorithm returned null, key: {0}, algorithm {1}.";
         public const string IDX10673 = "IDX10673: CryptoProviderFactory.GetHashAlgorithm returned null, factory: {0}, algorithm: {1}.";
         public const string IDX10674 = "IDX10674: JsonWebKeyConverter does not support SecurityKey of type: {0}";
-        public const string IDX10675 = "IDX10675: The value of '{0}' must be '{1}' bits, but was {2}.";
-        public const string IDX10676 = "IDX10676: ECDsa creation is currently only supported for the Windows operating system.";
```

Comment:
>IDX10676 [](start = 40, length = 8)

I suppose removing this could be a breaking change. Would it make sense somehow to mark it obsolete? or otherwise indicate that it's not used anymore?

Reply:
LogMessages is an internal class so this should not be a breaking change.

Reply:
people could have taken a dependency on the exception identifier

Reply:
No exception with the log identifier IDX10676  is thrown anymore, so nothing should break even if someone is taking a dependency on it.

On the other side, reusing old, deleted, log identifiers might be problematic. For that reason, we'll keep a deleted log message commented out, so we don't accidentally reuse one of the old identifiers when creating a new one.
```csharp
//public const string IDX10676 = "IDX10676:"

Reply:
I see your point. Our tooling reports if a log message is no longer used anywhere, so I guess that the decision to remove unused log messages is made before the tooling was setup.

The reason why we decided to comment-out a log message, instead of deleting, is to be able to setup our tools to search for duplicate log ids, even if a log message is commented-out. With that in place, we'll be sure that we are not using reusing old/deleted log identifiers.
Log message text is removed only to indicate that a particular log message considered as 'deleted'.